variables.tf

```diff
@@ -77,6 +77,25 @@ variable "minio_service" {
   description = "Minio data-object contains service_name, port, access_key and secret_key"
 }
 
+variable "minio_vault_secret" {
```

Comment:
Update readme section Inputs

Reply:
resolved in https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/53/commits/48107b09b5abb90d42a3c727b8572e543f3aa0da